File: Runners/JavaRunner/java-runner/src/main/java/com/mems/Shared/DTOs/CodeResponseDto.java
package com.mems.Shared.DTOs;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.mems.Shared.Enums.RequestStatus;
import java.util.UUID;

public class CodeResponseDto {
    @JsonProperty("RequestId")
    public UUID RequestId;

    @JsonProperty("Status")
    public RequestStatus Status = null;

    @JsonProperty("VersionId")
    public UUID VersionId;

    @JsonProperty("UserId")
    public UUID UserId;

    @JsonProperty("UserSolution")
    public String UserSolution;

    @JsonProperty("Language")
    public String Language;

    @JsonProperty("Result")
    public ExecutionResultDto Result;
}

File: Runners/KotlinRunner/src/main/kotlin/dto/CodeResponseDto.kt
package dto

import com.fasterxml.jackson.annotation.JsonProperty
import enums.RequestStatus
import java.util.UUID

data class CodeResponseDto(
    @field:JsonProperty("RequestId")
    var RequestId: UUID? = null,

    @field:JsonProperty("Status")
    var Status: RequestStatus? = null,

    @field:JsonProperty("VersionId")
    var VersionId: UUID? = null,

    @field:JsonProperty("UserId")
    var UserId: UUID? = null,

    @field:JsonProperty("UserSolution")
    var UserSolution: String? = null,

    @field:JsonProperty("Language")
    var Language: String? = null,

    @field:JsonProperty("Result")
    var Result: ExecutionResultDto? = null
)
